code/controllers/configuration/entries/general.dm

```diff
@@ -205,6 +205,13 @@
 
 /datum/config_entry/flag/use_exp_restrictions_admin_bypass
 
+/datum/config_entry/flag/use_low_living_hour_intern
```

Comment:
waste of a config entry which will result in this system being off by default, have it disable if use_low_living_hour_intern_hours is 0

Reply:
use_low_living_hour_intern is a flag due to the setup I used. It will feed from either use_low_living_hour_intern_hours or if that is not set use_exp_restrictions_heads_hours instead. Finally falling back through to a hardcoded value if necessary.